guides/defensive-domain-registration.md

```diff
@@ -0,0 +1,49 @@
+---
+expires: 2021-06-01
+---
+# Defensive domain registrations
+
+The MOJ and associated organisations (Executive agencies, non-departmental public bodies and so on) main varying levels of 'online presence' using domain registrations that are often a cornerstone of the organisation's identity on the public internet, for example, as the email domain for contacting other government organisations, partners and members of the public.
```

Comment:
main -> maintain
registrations that are often a cornerstone of -> registrations. This are a fundamental part of
internet, for example, as the email -> internet. An example is the `justice.gov.uk` email